pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -561,6 +575,58 @@ func (r *ReconcileLogStorage) Reconcile(ctx context.Context, request reconcile.R
 			return reconcile.Result{}, nil
 		}
 
+		kibanaInternalCertSecret, err := utils.GetSecret(ctx, r.client, render.KibanaInternalCertSecret, rmeta.OperatorNamespace())
+		if err != nil {
+			reqLogger.Error(err, err.Error())
+			r.status.SetDegraded("Waiting for internal Kibana cert secret to be available", "")
+			return reconcile.Result{}, err
+		}
+
+		esInternalCertSecret, err = utils.GetSecret(ctx, r.client, relasticsearch.InternalCertSecret, render.ElasticsearchNamespace)
+		if err != nil {
+			reqLogger.Error(err, err.Error())
+			r.status.SetDegraded("Waiting for internal Elasticsearch cert secret to be available", "")
+			return reconcile.Result{}, err
+		}
+
+		kubeControllersGatewaySecret, err := utils.GetSecret(ctx, r.client, render.ElasticsearchKubeControllersUserSecret, common.CalicoNamespace)
```

Comment:
So I think this logic is a little backwards, kubecontrollers should not be creating this secret, the logstorage controller should be creating this secret. It should put it in the operator namespace, and kubecontrollers should copy it over to it's namespace if it exists.

Previously that's how this was done, except it wasn't just a simple copy, we'd create a new secret with same values of the admin secret.